src/routes/api/articles/likeController.js

```diff
@@ -0,0 +1,101 @@
+import models from '../../../models';
+
+const {
+  likes
+} = models;
+
+/**
+ *
+ *
+ * @export
+ * @class likes
+ */
+export default class Likes {
+  /**
+ *
+ *
+ * @static
+ * @param {Request} request
+ * @param {Response} response
+ * @param {next} next
+ * @memberof Articles
+ * @returns {void}
+ */
+  static async likeArticle(request, response, next) {
+    try {
+      let option = true;
+      if (request.params.option === 'dislike') { option = false; }
+      const [like, isCreated] = await likes.findOrCreate({
+        where: { articleSlug: request.params.slug, userEmail: request.user.email },
+        defaults: { like: option },
+      });
+      if (isCreated) {
+        await Likes.countOptions(request, response, option);
+        return;
+      }
+      if (like.dataValues.like === option) {
+        await Likes.removeOption(like);
+        await Likes.countOptions(request, response, option);
+        return;
+      }
+      await Likes.changeOption(like, option);
+      await Likes.countOptions(request, response, option);
+    } catch (err) {
+      return next(err);
+    }
+  }
+
+  /**
+ *
```

Comment:
add a description of how this method works

Reply:
Thank you Kossy for the review
However, I do not fully understand what you want me to do.
Please kindly explain

Thanks